example/example_client.go

```diff
@@ -85,7 +85,26 @@ func (s udpDNSReportSender) Send(r choir.Report) error {
 	if err != nil {
 		log.Fatal("Failed to reach resolver:", err)
 	}
-	c.Write(query)
+	if _, err := c.Write(query); err != nil {
```

Comment:
nit: `defer c.Close()` and return early to reduce nesting below.

Reply:
Done